src/reducers/windowHandler.js

```diff
@@ -87,26 +86,17 @@ export default function windowHandler(state = initialState, action) {
                 })
             });
 
+        case types.CLOSE_PROCESS_MODAL:
+            if (state.modal.modalType === 'process') {
+                return Object.assign({}, state, {
+                    modal: Object.assign({}, state.modal, initialState.modal),
+                });
+            }
+            return state;
+
         case types.CLOSE_MODAL:
             return Object.assign({}, state, {
-                modal: Object.assign({}, state.modal, {
-                    visible: false,
-                    tabId: null,
-                    rowId: null,
-                    viewId: null,
-                    viewDocumentIds: null,
-                    layout: {},
-                    data: [],
-                    rowData: {},
-                    isAdvanced: false,
-                    title: '',
-                    type: '',
-                    modalType: '',
-                    saveStatus: {},
-                    validStatus: {},
-                    includedTabsInfo: {},
-                    triggerField: null
-                })
+                modal: Object.assign({}, state.modal, initialState.modal),
```

Comment:
Better: use

```js
modal: {
    ...state.modal,
    ...initialState.modal
}
```

We can do a refactoring specifically to replace all `Object.assign`s.

Reply:
I kept it for consistency, but you are right we should use this in all occurences instead as we use babel's `object-rest-spread` already